src/adagrams.js

```diff
@@ -1,8 +1,167 @@
+import _ from 'lodash';
+// var _ = require('lodash');
+
+    //letters/frequencies object
+    const LETTERS = {A : 9, B : 2, C : 2, D : 4, E : 12, F : 2, G : 3, H : 2, I : 9, J : 1, K : 1, L : 4, M : 2, N : 6, O : 8, P : 2, Q : 1, R : 6, S : 4, T : 6, U : 4, V : 2, W : 2, X : 1, Y : 2, Z : 1 };
+
+      //method to remove specific value from array
+  function removeItemsOnce(array, value) {
+    let index = array.indexOf(value);
+    array.splice(index, 1);
+    return array;
+  };
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    //Letters will now be populated for game play! Start with an empty array. Pulling from const LETTERS hash.
+    let letters = [];
+
+    //populate array with the correct number of letters (frequencies)
+    for (let [key] in LETTERS) {
+      for(let i = 1; i <= LETTERS[key]; i++) {
+        letters.push(key);
+        
+      }
+    }
+    //create a 'sample' method to return a sampling of tiles
+    const randomTenTiles = _.sampleSize; //_.sampleSize is lodash npm
+    let drawn = randomTenTiles(letters, 10);
+    return drawn;
+  },
+
+  //input = word played
+  usesAvailableLetters(input, lettersInHand) {
+    let playedTiles = input.toUpperCase().split('');
+    let dupLettersInHand = Array.from(lettersInHand);
+    // console.log(playedTiles);
+    if (playedTiles.length <= lettersInHand.length) {
+      for (let letter of playedTiles) {
+        if (dupLettersInHand.includes(letter)) {
+          removeItemsOnce(dupLettersInHand, letter); //calling function to delete tile from copied array. 
+        }
+        else {
+          return false;
+        };
+      };
+      return true;
+      
+        return false;
```

Comment:
This line is unreachable because it's right below another return, just like it would be in Ruby